sdks/python/apache_beam/examples/inference/pytorch_image_classification.py

```diff
@@ -122,10 +122,14 @@ def run(
     model_class = models.mobilenet_v2
     model_params = {'num_classes': 1000}
 
+  class PytorchModelHandlerTensorWithBatchSize(PytorchModelHandlerTensor):
```

Comment:
@yeandy I added batch size for this example. For language_modeling.py its already added as 1 due to `torch.stack'. I am leaving it as it is for now till we add the optional batching flag